devel/contents.tex

```diff
@@ -48,10 +48,16 @@ \section{Examination}%
 The report will be graded according to the following grading criteria:
 \begin{description}
   \item[Grade E] You fulfil all the \acp{ILO} at the minimum level.
+
+  \item[Grade C] You fulfil all the \acp{ILO},
+    you evaluations and designs are \emph{good} with \emph{some base} in theory 
```

Comment:
\emph{good} is very subjective, can it be written a bit more concrete? Again if we establish what the minimum expectancy is, this should be easier to define as well. Are we expecting the students to perform a "proper" literature study, where they have identified key articles in their chosen subject? What expectancies should we have on their application of the articles? Is it enough that they replicate them? Should they adapt them to a new usage area?
 When have a solution \emph{some base} in theory? Are we allowing the students to make some assumptions about the theory?

Reply:
They should define a problem, e.g. improve logins to websites from a usability and security perspective (or as one asked "why can't we have as easy sign in to websites as for the phone?"). They should outline a solution or path towards a solution to this problem.

Then they should look for literature where people have tried to solve this problem, some have focused on usability, some on security. Some things can be applied to their problem some cannot, evaluation is what brings insights about this to the students.

Then they can suggest that some results can (probably) be combined to solve the problem. This suggestion can be more or less founded in theory (papers, stuff from the course, their evaluations).

So: No, they don't have to do a literature study. No, they don't have to apply articles, maybe there are none focused on their problem.

But your question about replication is interesting: what should happen if there is a paper solving their exact problem? Obviously there will be a problem for them to show their abilities if they just copy the paper. So I suppose they have a few options: (1) change the problem, (2) thoroughly analyse the paper, find downsides and suggest improvements. I suppose (1) sort of covers adapt to a new usage area.

Reply:
You should have identified a relevant problem, and given a solution to it.
It must be a viable solution, gaps and errors are allowed, if they only would render your solution less optimal.

Reply:
\item[Grade A] You should have identified a relevant problem, and given a solution to it.
It must be a viable solution, gaps and errors are not allowed unless they have been properly addressed and you have given a suggestion on an approach to how to start solve the issue.

Reply:
Not sure if this version of C and A improves the situation. In particular I miss "\emph{well-founded/some base} in theory and, where applicable, the research literature".